lib/usd/translators/jointWriter.cpp

```diff
@@ -286,11 +295,12 @@ static bool _GetJointLocalRestTransformsFromDagPose(
     // Use whatever bindPose the root joint is a member of.
     MObject bindPose = _FindBindPose(rootJoint);
     if (bindPose.isNull()) {
-        TF_WARN(
-            "%s -- Could not find a dagPose node holding a bind pose: "
-            "The Skeleton's 'restTransforms' property will not be "
-            "authored.",
-            skelPath.GetText());
+        TF_DEBUG(PXRUSDMAYA_TRANSLATORS)
```

Comment:
Interested to know why we want to switch this warning to a debug trace? (and same for the one below)

Reply:
Because we now have an additional fallback - we set the `restTransforms` from the bindPose - whereas before this would mean that we would "give up", and not author any `restTransforms`.

I don't mind changing either this or the next one back to `TF_WARN` if you want though.

Actually - on looking again, I think I'll definitely change the next one back to `TF_WARN`, because the inability to read the local transform shouldn't happen.  (Whereas with this one, it merely indicates that there was no dagPose... which, while not standard, is still kosher.)

Reply:
Changed the second to `TF_WARN`, left this one as `TF_DEBUG` for now - let me know if you want this one back to `TF_WARN` as well:
https://github.com/Autodesk/maya-usd/pull/1130/commits/afaea5505a8aadef0d02bff165e72cc36e5c8a36